flathub.json

```diff
@@ -0,0 +1,4 @@
+{
+    "end-of-life": "App renamed to: org.luanti.luanti. Read: https://github.com/flathub/net.minetest.Minetest/issues/5",
```

Comment:
btw i don't think it matters much but maybe a more user-friendly message is possible? not sure how i would word it though.

Reply:
Please, I would like a suggestion for a message, could you give me some?

Reply:
I think that's probably fine, yeah

Reply:
Made the changes